Source/WebKit/UIProcess/ios/fullscreen/WKFullScreenViewController.mm

```diff
@@ -846,6 +909,7 @@ - (void)loadView
     [_banner setTranslatesAutoresizingMaskIntoConstraints:NO];
 
     [_animatingView addSubview:_banner.get()];
+#endif
```

Comment:
All of this seems to be common code between the glass effect vs legacy paths. Is there any way you could avoid the duplication?

```objc
    _banner = WTFMove(banner);

    _bannerTapToDismissRecognizer = adoptNS([[UITapGestureRecognizer alloc] initWithTarget:self action:@selector(_bannerDismissalRecognized:)]);
    [_bannerTapToDismissRecognizer setDelegate:self];
    [_banner addGestureRecognizer:_bannerTapToDismissRecognizer.get()];

    [_banner setTranslatesAutoresizingMaskIntoConstraints:NO];

    [_animatingView addSubview:_banner.get()];
```